### DISCUSSION TITLE
benchmarking depth

### DISCUSSION THREAD

**Discussion by JM-Bader** [vdemichev/DiaNN#520] [Q&A]
To benchmark numbers, the  DIA-NN github recommends to use the relaxed protein inference ( --relaxed-prot-inf command or ticking the heuristic protein inference button in the GUI): 

![image](https://user-images.githubusercontent.com/98888664/194027954-0789b4f1-cd89-4b98-85df-c60cde9ea1c1.png)

The proteome data that should be used for differential expression and other downstream analyses (taking care with GO enrichment etc.) should be the non-heuristic protein grouping. 

Q1: So should one report the protein group numbers from the heuristic search (in a publication) and use the non-heuristic search results for actual analysis? 
Q2: What about only performing the non-heuristic search and reporting both the number of protein groups and the number of gene groups ?
Q3: What is the difference between gene groups (non-heurisitic search) and heuristically-inferred protein groups ? 
Q4: Where to get accurate gene group information from ? According to other threads, the main report should be used and not the pg.matrix report. So likewise, should the gg.matrix not be used? 
Q5: How to extract the gene groups from the main report?

**Comment by vdemichev** [comment]
This part of the readme is a 'remnant' of DIA-NN 1.8. In 1.8.1 --relaxed-prot-inf is default ('Heuristic protein inference' checkbox). For DE heuristic is still OKish, other tools & their users seem to be fine with that, so can use it. But yes, non-heuristic is potentially somewhat more reliable.
Q1. Numbers - only heuristic search.
Q2. Does not make sense.
Q3. Gene groups are just sets of genes corresponding to protein in protein groups, in either case.
Q4. Matrices are there for users which don't use R or Excel. In all other cases better use the main report, as it gives one a clear understanding of how the data is filtered, etc. gg_matrix is fine if you are fine with its filtering.
Q5. diann_matrix(group.header = Genes, quantity.header = Genes.MaxLFQ)?
